StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -100,6 +84,45 @@ StETofNode::StETofNode( const TGeoPhysicalNode& gpNode, const float& dx, const f
     buildMembers();
 }
 
+StETofNode::StETofNode( const TGeoPhysicalNode& gpNode, const float& dx, const float& dy, const StThreeVectorD alignment )
```

Comment:
Agreed. This is part of the core for this class that was written by Florian and it seems his logic seemed to be to pass just about everything as references. I also do not see it making a difference (even if someone were to change dx/dy in this method accidently, it would not matter, as these parameters are passed once and then never used again. ). I could go through the full file and change that consistently, but i see little benefit from it. What do you think?

Reply:
Although it is considered to be a good practice, I don't insist because practically this is unlikely to result in any noticeable difference. But if you do decide to make these changes please do that in a separate PR.